src/platform/plugins/shared/data/public/actions/filters/create_filters_from_value_click.ts

```diff
@@ -226,20 +226,32 @@ export const appendFilterToESQLQueryFromValueClickAction = ({
   if (!dataPoints.length) {
     return;
   }
-  const { table, column: columnIndex, row: rowIndex } = dataPoints[dataPoints.length - 1];
 
-  if (table?.columns?.[columnIndex]) {
-    const column = table.columns[columnIndex];
-    const value: unknown = rowIndex > -1 ? table.rows[rowIndex][column.id] : null;
-    if (value == null) {
-      return;
+  let queryString = query.esql;
```

Comment:
rather than creating the filter just for the last datapoint passed in the action, create the filter using all datapoints (to mimic behaviour in DSL mode)